lib/tools/Curl.class.php

```diff
@@ -45,13 +48,28 @@ public static function doCurl($method, $url, $params = [], $options = [])
       throw new DomainException('Invalid curl options: ' . join(', ', array_keys($invalid)));
     }
 
-    $options = array_merge($defaults, $options);
-
     if (!in_array($method, [static::GET, static::POST, static::PUT]))
     {
       throw new DomainException('Invalid method: ' . $method);
     }
 
+    $options = array_merge($defaults, $options);
+
+    if (!Apc::isEnabled())
+    {
+      $options['cache'] = false;
+    }
+
+    if ($options['cache'])
```

Comment:
I don't think this cache stuff belongs in the Curl class. happy to discuss if you disagree.